doc/source/whatsnew/v0.20.0.txt

```diff
@@ -122,6 +122,16 @@ Other enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _whatsnew.api_breaking.io_compat
+
+Pickle and HDF5 compat requires >= 0.13.0
```

Comment:
no the issue is that if a TimeSeries is used it won't work (and that's why I deleted the pickle files < 0.13.0). To be honest I don't have a problem with just saying we are pickle back-compat to >= 0.13. 

It still will work if TimeSeries is not present though.